### DISCUSSION TITLE
Telnet Star Wars

### DISCUSSION THREAD

**Discussion by Mulhollandrl** [cc-tweaked/CC-Tweaked#798] [Q&A] [answered]
I know that the old Computer Craft had the Telnet starwars in it, and I was wondering if this one did too. I looked through the files, and there was a secret disk file called treasure_disk.json, but I have no clue if that is what I think it is.

Thank you for your time!

**Comment by Lupus590** [comment, marked as answer]
https://github.com/SquidDev-CC/treasure-disks/tree/master/data/computercraft/lua/treasure/dan200/alongtimeago

They have moved out a while ago and deprecated a lot longer than that.
